### ISSUE TITLE
Image upload using rtMedia causes "BuddyPress Edit Activity" plugin icon to disappear

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtCamp/rtMedia#1681]
```
Test Environment
rtMedia: v4.6.4
BuddyPress: v6.1.0
BuddyPress Edit Activity: v1.1.1
```

Steps to Reproduce
- Install and activate the plugins mentioned in test environment
- Go to activity page and add a activity without media
- Notice that with no media if activity is posted then `Edit` option appears for that post
- But, when media is uploaded in the activity the `Edit` option doesn't appear

**Comment by pooja-muchandikar** [comment]
@SID177 

Tested the issue with following test environment and the fix works as expected. :heavy_check_mark: 

```
Test Environment
rtMedia: v4.6.4
BuddyPress: v6.2.0 ( legacy and nouveau template )
BuddyPress Edit Activity: v1.1.1
```

**Comment by pooja-muchandikar** [comment]
Tested with `pre-release` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.5`
